pages/about-us.tsx

```diff
@@ -27,12 +28,18 @@ interface IAboutUs {
   texts: any
   team: teamType[]
   partner: string[]
+  error: string
 }
 
-const AboutUs = ({ texts, team, partner }: IAboutUs) => {
+const AboutUs = ({ texts, team, partner, error }: IAboutUs) => {
   const dispatch = useDispatch();
 
   useEffect(() => {
+    if(error){
+      dispatch(errorHandling({ isError: true }));
+    }else {
+      dispatch(errorHandling({ isError: false }));
+    }
     dispatch(changeData(texts));
   });
```

Comment:
```suggestion
    if(error){
      dispatch(errorHandling({ isError: true }));
    }else {
      dispatch(errorHandling({ isError: false }));
    }
    dispatch(changeData(texts));
  });
```

create a separate hook called useFetchedData(error, data) and move all this logic there and use overall in project.

Reply:
i can't use custom hooks in useEffect